ci/test/02_run_container.sh

```diff
@@ -30,6 +30,10 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
   docker volume create "${CONTAINER_NAME}_previous_releases" || true
 
   if [ -n "${RESTART_CI_DOCKER_BEFORE_RUN}" ] ; then
+    # Kill harder, prevents "The container name "/ci_..." is already in use by container"
+    docker stop "$(docker ps -a -q)" || true
+    sleep 3
```

Comment:
Also, why is this `sleep` here?

Reply:
Without that command I got the same behavior as if the command wasn't there. Presumably it's asynchronous and we somehow have to wait for it. Maybe the above solution avoids the problem altogether.

Reply:
I got rid of sleep, in favor of `docker rm -f $CONTAINER_NAME` (haven't pushed that change yet, pending discussion on podman)

https://stackoverflow.com/questions/57631654/how-to-wait-for-a-docker-container-to-be-removed

Reply:
These lines are dropped now.